search_that_hash/__main__.py

```diff
@@ -3,12 +3,13 @@
 import toml
 
 import click
-from loguru import logger
 
 from search_that_hash.cracker import cracking
 from search_that_hash import config_object
 from search_that_hash import printing
 
+import logging
+import coloredlogs
```

Comment:
Is this a dependency not defined in `pyproject.toml`?

Reply:
Will add coloredlogs to it now